src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -83,7 +93,12 @@ object ConsignmentFields {
     Field("getConsignment", OptionType(ConsignmentType),
       arguments = ConsignmentIdArg :: Nil,
       resolve = ctx => ctx.ctx.consignmentService.getConsignment(ctx.arg(ConsignmentIdArg)),
-      tags = List(ValidateUserOwnsConsignment(ConsignmentIdArg))
+      tags = List(ValidateUserHasAccessToConsignment(ConsignmentIdArg))
+    ),
+    Field("getConsignmentExport", OptionType(ConsignmentType),
+      arguments = ConsignmentIdArg :: Nil,
```

Comment:
Do we need another field here. The new `ValidateUserHasAccessToConsignment` checks for export access so couldn't we just use the existing `getConsignment` field?

Reply:
Can do. Just means this will pull back all the additional fields needed for the export, but not necessarily used for anything else.

Reply:
But the existing `getConsignment` query won't be requesting those fields and you can do a new query in the `tdr-generated-graphql` project like
```
query getConsignmentForExport($consignmentId: UUID!) {
    getConsignment(consignmentid: $consignmentId) {
        userid
        seriesid,
        datetime,
        transferInitiatedDatetime,
        exportDatetime      
    }
}
```
That keeps things separate without needing the new field.

Reply:
Yeah, realised that after I wrote that comment!